### PR TITLE
Improve the code to make the order of the model's fields consistent w…

### PR DESCRIPTION
**Pull request by zhangxiaodel**
Improve the code to make the order of the model's fields consistent with the order in the table ddl

### CODE REVIEW

File: utils_mysql.go [hunk 1]
```diff
@@ -40,38 +41,40 @@ func GetColumnsFromMysqlTable(mariadbUser string, mariadbPassword string, mariad
...
-func generateMysqlTypes(obj map[string]map[string]string, depth int, jsonAnnotation bool, gormAnnotation bool, gureguTypes bool) string {
+func generateMysqlTypes(obj map[string]map[string]string, columnsSorted []string, depth int, jsonAnnotation bool, gormAnnotation bool, gureguTypes bool) string {
 	structure := "struct {"
 
-	keys := make([]string, 0, len(obj))
-	for key := range obj {
-		keys = append(keys, key)
-	}
-	sort.Strings(keys)
+	//keys := make([]string, 0, len(obj))
```
Comment (Shelnutt2) [inline utils_mysql.go]: Go ahead and remove these sections since now you pass in the sorted columns

**Review by Shelnutt2 — COMMENTED** [review]
@zhangxiaodel thank you for your contribution. If you won't mind address the one minor comment, then lgtm.

File: utils_mysql.go (hunk 1 above)
Comment (zhangxiaodel) [reply utils_mysql.go]: ok, This project helped me a lot , thank you

Comment (Shelnutt2) [reply utils_mysql.go]: Thanks!

**Pull request merged**